app/views/transient_registrations/show.html.erb

```diff
@@ -302,6 +243,48 @@
             </tbody>
           </table>
         <% end %>
+
+
+        <% if @transient_registration.main_people.any? %>
+          <h2 class="heading-medium">
+            <%= t(".person_information.heading.#{@transient_registration.business_type}") %>
+          </h2>
+
+          <table>
+            <tbody>
+              <tr>
+                <th>
+                  <%= t(".person_information.name") %>
+                </th>
+                <th>
+                  <%= t(".person_information.dob") %>
+                </th>
+                <th>
+                  <%= t(".person_information.convictions") %>
+                </th>
+              </tr>
+              <% @transient_registration.main_people.each do |person| %>
+                <tr>
+                  <td>
+                    <%= person.first_name %> <%= person.last_name %>
+                  </td>
+                  <td>
+                    <%= person.dob.to_date %>
+                  </td>
+                  <td>
+                    <% if person.conviction_search_result.present? %>
```

Comment:
I think all people will have a `conviction_search_result` - it's whether the `match_result` is positive or negative that we want to check.

The only exception is if the application is still being filled in and the check hasn't run yet.

Reply:
Oh I see!! Thanks! :)

Reply:
```suggestion
                    <% if person.conviction_check_required? %>
```